src/objectExplorer/objectExplorerService.ts

```diff
@@ -319,6 +317,20 @@ export class ObjectExplorerService {
         }
     }
 
+    private createNodeLabel(credentials: IConnectionCredentials): string {
+        const database = credentials.database;
+        const server = credentials.server;
+        if (database && database !== '') {
+            return `${server} (${database})`;
+        } else {
```

Comment:
You should include auth type/username as well

Reply:
We don't do that in ADS.

Reply:
Yes we do

![image](https://user-images.githubusercontent.com/28519865/65711496-da3a5080-e049-11e9-871b-50fbda23d548.png)

Reply:
oops